_script/cibuild

```diff
@@ -13,6 +13,6 @@ if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" = "master" ]
 then
     echo "INFO: Merging to master... time to build and deploy redistributables"
     docker_name="dachad/tcpgoon"
-    ./_script/build "$docker_name"
+    source ./_script/build "$docker_name"
```

Comment:
i think now I know why you were exporting the commit hash, and why you may be sourcing this script rather than just executing it (probably to reuse the global var COMMIT_HASH). You have to think about another way. This is a "dirty hack", my friend. Good try... i wait for your new proposal... if you don't find the way (probably yes), I will share an idea ;)

Reply:
@dcaba I tried another way around in commit f0becf0
if there is a better way, please, enlighten me :)